install.rb

```diff
@@ -294,6 +308,12 @@ def prepare_installation
     bindir = RbConfig::CONFIG['bindir']
   end
 
+  if not InstallOptions.localedir.nil?
+    localedir = InstallOptions.localedir
+  else
+    localedir = "/opt/puppetlabs/puppet/share/puppet/locale"
```

Comment:
Are translations really "configuration"? I would not expect them to be user-modifiable or user-replaceable. We ship them with a given version of the project and that's it. In my mind that data belongs in "Program Files"